app/models/alerts.py

```diff
@@ -17,10 +17,43 @@ class Alerts(SerialisedModelCollection):
     def current_and_public(self):
         return [alert for alert in self if alert.is_current_and_public]
 
+    @property
+    def planned(self):
+        return [
+            alert for alert in self if alert.is_planned and not alert.is_public
+            ] + [
+                planned_test for planned_test
+                in self.planned_tests
+                if planned_test.is_planned
+            ]
+
```

Comment:
It would be good to have a test for this

Reply:
Have added tests for the is_planned property of the alert model and a few for this one, too.